Enumerables/enumerables.rb

```diff
@@ -0,0 +1,139 @@
+module Enumerable
+  # 1
+  def my_each
+    return to_enum unless block_given?
```

Comment:
🤔 This will always return to_enum (unless a block is given). Shouldn't it return `.size` ?

Reply:
Ok but, the original each without block always return an enumerator.
Here es an example: 

![Screenshot 2019-03-21 at 12 32 46 PM](https://user-images.githubusercontent.com/1456388/54772587-aaf7e200-4bd5-11e9-976e-d3e1f1db9f23.png)